src/solvers/refinement/string_refinement.cpp

```diff
@@ -606,7 +608,7 @@ decision_proceduret::resultt string_refinementt::dec_solve()
   {
     bool satisfied;
     std::vector<exprt> counter_examples;
-    std::tie(satisfied, counter_examples)=check_axioms(
+    std::tie(satisfied, counter_examples) = check_axioms(
```

Comment:
Whitespace change?

Reply:
Function arguments change triggered reformatting of this line.

Reply:
Still can't spot it.

Reply:
Lines 620 and 621

Reply:
Sure, but this is line 609/611. Can't see how that's related. Similarly for the other one (669 and 670 vs. 657/660). I don't get it.

Reply:
Formatting those lines with clang format formats this line. I tried changing it, and clang-format rolled it back again to ` = `.

Reply:
`git add -p` is your friend.

Reply:
It doesn't matter how I add those changes. It won't go through CI.

Reply:
My `.travis.yml` has `formatting-stage` marked as `allow_failures`. If you insist to be strict about those, put whitespace changes into a separate commit containing only those.

Reply:
As requested, I created another "clang format" commit.

Reply:
Thank you!